docs/operate/reference/viam-server/debug-endpoints.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Debug Endpoints"
+linkTitle: "Debug Endpoints"
+weight: 130
+type: "docs"
+description: "Advanced debugging endpoints available in viam-server for troubleshooting and development."
+date: "2025-05-19"
+---
+
+The RDK (Robot Development Kit) that powers `viam-server` includes several debug endpoints that expose server internals.
+These endpoints can be helpful during development or when troubleshooting complex issues:
+
+- [`pprof`](#pprof)
+- [`graph`](#graph)
+
+## `pprof`
+
+Setting `enable_web_profile` to `true` in your machine configuration allows you to visualize runtime profiling data in the localhost browser.
+The pprof tool provides valuable insights into your application's performance, including:
+
+- CPU usage
+- Memory allocation
+- Goroutine blocking
+- Execution tracing
+
+This information can be particularly useful when:
+- Diagnosing performance bottlenecks
+- Investigating memory leaks
+- Understanding resource usage patterns
+- Troubleshooting high CPU or memory usage
```

Comment:
```suggestion
```

Duplicated information from the troubleshooting page. I don't think it adds any value here; anyone who's reading this page should already know the value of this information!